Makes Disassembler as a binary analyzer (#166)
obhq/obliteration: README.md

- mac-release
- mac-debug

> If all you want is using the emulator, choose `[YOUR-PLATFORM]-release` for optimized outputs. But if you want to edit the code, choose `*-debug`.

Reviewer: Minor grammar suggestions

```suggestion
If all you want is to use the emulator, choose `[YOUR-PLATFORM]-release` for optimized outputs. But if you want to edit the code, choose `*-debug`.
```

### Build

```sh
cmake --build build
```

You can use `-j` to enable parallel build (e.g. `cmake --build build -j 2`). Each parallel build consume a lot of memory so don't specify the number of your CPU cores otherwise your system might be crashing due to out of memory.

## Development

> Before proceed, make sure the build preset you was using is `*-debug`. We recommended Visual Studio Code as a code editor with the following extensions:

Reviewer: ```suggestion
Before proceeding, make sure the build preset you are using is `*-debug`. We recommend Visual Studio Code as a code editor with the following extensions:
```

- [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools)
- [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer)
- [CMake](https://marketplace.visualstudio.com/items?itemName=twxs.cmake)
- [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools)
- [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)

> Then open this folder with VS Code. It will ask which CMake preset to use and you need to choose the same one that you was using when building. Everything should work out of the box (e.g. code completion, debugging, etc).

Reviewer: ```suggestion
Then open this folder with VS Code. It will ask which CMake preset to use and you need to choose the same one that you were using when building. Everything should work out of the box (e.g. code completion, debugging, etc).
```
